docs/product_release.md

```diff
@@ -0,0 +1,18 @@
+# Product Release
+
+## Artifacts
+
+- Container images
+- Helm Chart
+- GitHub release (Changelog + Sourcecode)
+
+## Versioning
+
+- Semantic Versioning
+
+## Quality Assurance
+
+- Defined in TRGs
+- Responsibility of every contributor
+- (Additonally verified in quality gate process. TBD, if kept after consortia)
+
```

Comment:
## process abstract ##
Once planning activities for a certain iteration are completed: all contributors follow their state-of-the-art feature work on all issues in status `work in progress`. The vehicle of product helm charts shall be used to integration-test any dependencies. All valid TRGs shall be followed, as those are crosschecked by committers upon merging any pull request - leading into a flawless GitHub release per product.

Reply:
Not quite sure, but AFAIK, only overarching features on the dataspace level are planned in `sig-release` and therefore are handled on a board with `work in progress` status. 
Product features, that support the dataspace feature, could (and IMO should) be planned on product repository level. So far, we do not define rules for how products need to handle issues and boards

Reply:
update @stephanbcbauer & @RolaH1t 21-Mar:
Once planning activities for a certain iteration are completed: all contributors follow their state-of-the-art feature work on all issues in status work in progress. The vehicle of product helm charts shall be used to integration-test any dependencies. All valid TRGs must be followed, as those are crosschecked by committers upon merging any pull request - leading into a flawless GitHub release per component.

Reply:
"state-of-the-art feature work" sounds a bit bloated, don't you think? What does that even mean 😁?

Reply:
@RolaH1t and @stephanbcbauer,
I took your input and did elaborate an change wording. What do you think of the description in this commit
[c33fcc8](https://github.com/eclipse-tractusx/sig-release/pull/519/commits/c33fcc82d867d7bbd776cd09e20130344b0129da)?